pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -321,9 +322,6 @@ public BrokerService(PulsarService pulsar, EventLoopGroup eventLoopGroup) throws
                 .newSingleThreadScheduledExecutor(new DefaultThreadFactory("pulsar-stats-updater"));
         this.authorizationService = new AuthorizationService(
                 pulsar.getConfiguration(), pulsar().getPulsarResources());
-        if (!pulsar.getConfiguration().getEntryFilterNames().isEmpty()) {
```

Comment:
it is better still keep a list of global entry filters and initialise them only once.

in a broker with thousands of topics the overhead may be non negligible, a EntryFilterFactory may have to perform some initialisation at boot  

also a EntryFilter may have some local cache.
in 2.10 the EntryFilter instance is global, so the cache is shared, if we create a filter entry per each topic then this kind of caching will be useless.

I think that we have to keep a list of EntryFilters globally, per-namespace and per-topic.

Reply:
I remove this, because it's not used any more.